packages/tx/src/types.ts

```diff
@@ -152,6 +171,32 @@ export interface TransactionInterface<T extends TransactionType> {
   errorStr(): string
 }
```

Comment:
What is somewhat missing here which would complement the picture would be a `LegacyTxInterface`.

I know this is likely the hardest one with eventual conflicts on re-/cross-used functionality or property access.

But did you try? 

That would be great if we would/could achieve that. Since here we - again - at least to some extend replicate this old hierarchy structure which we would optimally want to completely leave behind.

Reply:
Good point. Added a LegacyTxInterface, makes this more consistent and complete.